packages/bubblewrap.rb

```diff
@@ -3,17 +3,34 @@
 class Bubblewrap < Package
   description 'bubblewrap works by creating a new, completely empty, mount namespace'
   homepage 'https://github.com/containers/bubblewrap'
-  version '0.4.1_1'
+  @_ver = '0.4.1'
+  version "#{@_ver}-1"
   compatibility 'all'
-  source_url 'https://github.com/containers/bubblewrap/releases/download/v0.4.1/bubblewrap-0.4.1.tar.xz'
+  source_url "https://github.com/containers/bubblewrap/releases/download/v#{@_ver}/bubblewrap-#{@_ver}.tar.xz"
   source_sha256 'b9c69b9b1c61a608f34325c8e1a495229bacf6e4a07cbb0c80cf7a814d7ccc03'
 
+  binary_url({
```

Comment:
Somehow your binaries depend on dconf. Maybe try rebuilding them with dconf uninstalled?

`flatpak: error while loading shared libraries: libdconf.so.1: cannot open shared object file: No such file or directory`

Reply:
Is that bad? We could just add a dconf dependency..

Reply:
why not, we can add a dconf dependency.